scripts/patch-drizzle-kit-progress.mjs

```
@@ -0,0 +1,81 @@
+import { readFileSync, writeFileSync } from "node:fs";
+import { join, dirname } from "node:path";
+import { fileURLToPath } from "node:url";
+
+const root = join(dirname(fileURLToPath(import.meta.url)), "..");
+const binPath = join(root, "node_modules", "drizzle-kit", "bin.cjs");
+
+let bin;
+try {
+  bin = readFileSync(binPath, "utf8");
+} catch {
+  process.exit(0);
```

ESLint

[error] 'process' is not defined.

(no-undef)